src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -60,10 +61,21 @@
             <tag name="serializer.normalizer" />
         </service>
 
-        <!-- Resource path naming strategies -->
+        <!-- Resources Operations path resolver -->
+
+        <service id="api_platform.operation_path_resolver" alias="api_platform.operation_path_resolver.router" public="false" />
+
+        <service id="api_platform.operation_path_resolver.router" class="ApiPlatform\Core\Bridge\Symfony\Routing\RouterOperationPathResolver" public="false">
+            <argument type="service" id="api_platform.router" />
+            <argument type="service" id="api_platform.operation_path_resolver.custom" />
+        </service>
+
+        <service id="api_platform.operation_path_resolver.custom" class="ApiPlatform\Core\PathResolver\CustomOperationPathResolver" public="false">
```

Comment:
It would be weird to decorate a service defined by the user in case he uses its own path resolver, don't you think ?

About the `default` part, that's just an alias and i couldnt use `.operation_path_resolver` as it is already in use. Maybe a better name would be `.fallback` or `.root`?